AGPs/AGP-ScoutDAO.md

```diff
@@ -0,0 +1,83 @@
+---
+AGP: N/A
+Title: ScoutDAO
+Author: burrrata (@burrrata) and Joe Charlesworth (@joeycharlesworth)
+Status: Stage III
+Track: Finance
+Created: 2019-10-02
+---
+
+# AGP-X: ScoutDAO
+
+## Address of the transfer recipient
+
+ScoutDAO (DAO vault address)
```

Comment:
Please input the address of the DAO or the one person or entity that will provide the address